DESCRIPTION

```diff
@@ -1,14 +1,16 @@
 Package: washr
-Title: Efficient workflow tools for openwashdata package
+Title: R Data Package Development Tools For Openwashdata
```

Comment:
@mianzg: there is no need to capitalize each work. You can keep it lower case;

R data package development tools for openwashdata

Reply:
It has to be Title Case, required by CRAN

Reply:
washr: R Data Package Development Tools for Water Sanitation and Hygiene Data